nimble/host/syscfg.yml

```diff
@@ -497,3 +497,6 @@ syscfg.logs:
 
 syscfg.vals.BLE_MESH:
     BLE_SM_SC: 1
+
+syscfg.restrictions:
+    - BLE_TRANSPORT_HS == "native"
```

Comment:
hmm I'm not sure if I get this restriction

Reply:
this is to make sure transport on hs side is not overridden to something else when an actual host is included in build - such conbination would not make sense